meinberlin/apps/budgeting/assets/react_vote_button_init.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { widget as ReactWidget } from 'adhocracy4'
+import { VoteButton } from './VoteButton.jsx'
+
+function init () {
+  ReactWidget.initialise('mb', 'vote_button',
+    function (el) {
+      const props = el.getAttribute('data-attributes')
+      ReactDOM.render(<VoteButton {...props} />, el)
```

Comment:
somehow the passed props is a string, but should be an object.

Reply:
ok thats weird, will take a look

Reply:
> somehow the passed props is a string, but should be an object.

turned out, this does not happen.